api_specs/v3/component_analyses_v3.yaml

```diff
@@ -0,0 +1,413 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: '3.1'
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Dependency Analytics Server"
+security:
+- tokenAuth: []
+paths:
+  /component-vulnerability-analyses/:
```

Comment:
typo: it should be analysis not analyses

Reply:
ACK.I have however seen the use of "analyses" in many places in our code.

Reply:
actually somebody made a typo and people are following it. there is no word called "analyses" :)

Reply:
"analyses" has also been used in db schemas.